app/cdap/components/PipelineList/EmptyList/index.tsx

```diff
@@ -89,13 +89,13 @@ class EmptyList extends React.PureComponent<IProps, IState> {
         return;
       }
 
-      const fileDataString: string = evt.target.result;
+      let fileDataString: string = evt.target.result;
```

Comment:
Can you check if the error check requires the original `fileDataString`, if it doesn't care you can just combine the two lines into one

Reply:
Once original `fileDataString` is validated and doesn't have any errors, then only it is required to adjust the config node.